huaweicloud/services/dms/resource_huaweicloud_dms_rabbitmq_instance.go

```diff
@@ -247,6 +260,20 @@ func resourceDmsRabbitmqInstanceCreate(ctx context.Context, d *schema.ResourceDa
 	config.MutexKV.Lock(lockKey)
 	defer config.MutexKV.Unlock(lockKey)
```

Comment:
It seems that there is no global lock now, it is recommended to try to remove it.